Toolchain/Dockerfile

```diff
@@ -8,27 +8,24 @@ RUN apt-get update -y \
         ccache \
         cmake \
         curl \
+        e2fsprogs \
+        g++-10 \
```

Comment:
I think Ubuntu 21.10 ships with gcc 11 by default? At list given a cursory look at Google. So the default distro gcc should be fine to build the Toolchain and Lagom

Reply:
We don't have clear GCC version requirements in `Documentation/BuildInstructions.md`, so I guess that we indeed should default to whatever GCC is coming with the distribution.

Reply:
@ADKaster it seems that `Toolchain/BuildQemu.sh` fails to build Qemu 6.1.0 when using version 11, so we're not really able to go to that version yet.

Reply:
Oh really? That's pretty surprising. Mmk. Not a big deal either way